public/locales/es/translation.json

```diff
@@ -172,12 +172,12 @@
+  "tab5-p4": "<strong>Esto amplía quien puede solicitar beneficios del UI, debido al COVID-19.</strong> Esto incluye a dueños de negocio, contratistas independientes, personas que trabajan por su propia cuenta, trabajadores independientes o trabajadores de la economía colaborativa, y personas con historial de trabajo limitado. También está disponible para las personas que están cumpliendo con semanas de castigo en su solicitud o cuya solicitud fue considerada no ser elegible. PUA respalda las solicitudes presentadas entre el 2 de febrero y el 26 de diciembre de 2020. Lea más sobre la <3>Asistencia de Desempleo por la Pandemia</3>.",
   "tab5-header4": "Compensación Federal de Desempleo por la Pandemia  (FPUC) o Compensación Adicional por la Pandemia (PAC)",
-  "tab5-p5": "<strong>Esto proporciona $600 adicionales por semana</strong> para las personas que están recibiendo beneficios del UI. Es retroactivo para las solicitudes presentadas entre el 29 de marzo y el 25 de julio de 2020. Este beneficio de $600 se emite cada dos semanas. Lo recibirá al mismo tiempo que sus beneficios de UI o PUA.",
+  "tab5-p5": "<strong>Esto proporciona $600 adicionales por semana</strong> para las personas que están recibiendo beneficios del UI. Es retroactivo para las solicitudes presentadas entre el 29 de marzo y el 25 de julio de 2020. Lo recibirá al mismo tiempo que sus beneficios de UI o PUA, cada dos semanas.",
   "tab5-header5": "Compensación de Desempleo de Emergencia por la Pandemia (PEUC)",
-  "tab5-p6": "<strong>Esto le da 13 semanas adicionales de beneficios</strong> para las personas que están recibiendo beneficios del UI entre el 29 de marzo y el 26 de diciembre de 2020. California le da hasta 26 semanas de beneficios del UI dependiendo de la cantidad de sus ingresos durante el periodo de cuatro trimestres en los que se basa su solicitud. La Ley CARES alarga los beneficios a un total de 39 semanas.",
-  "tab5-blockquote1": "<strong>Nota:</strong> Aún estamos trabajando en la implementación del programa PEUC. Por favor continúe revisando nuestro sitio web para ver instrucciones sobre los próximos pasos para el programa PEUC.",
+  "tab5-p6": "<strong>Esto le provee 13 semanas adicionales de beneficios</strong> para las personas que se agotaron todos los beneficios de su solicitud del UI. La extensión de PEUC está disponible entre el 29 de marzo y el 26 de diciembre de 2020. California le da hasta 26 semanas de beneficios del UI. La Ley CARES extiende los beneficios a un total de 39 semanas.",
+  "tab5-blockquote1": "Para obtener más información, visite <3>Disposiciones de la Ley CARES</3>.",
```

Comment:
Can you change this to <2>Disposiciones de la Ley CARES</2>?  Otherwise, the link doesn't show up.  Sorry, it's not obvious how the links and numbers get assigned.

Reply:
Very helpful, thank you!